app/src/main/java/com/example/mylittlelibrary/MyLittleLibraryApplication.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.mylittlelibrary
+
+import android.app.Application
+
```

Comment:
Add application class for the project. URL:https://docs.sonarqube.org/display/SONARQUBE45/Developers%27+Seven+Deadly+Sins

Reply:
But... it's not doing anything yet.  It is overriding a function, and then invoking that function.